config/command_aliases.json

```diff
@@ -0,0 +1,3 @@
+{
+    "blacklist": "bl"

```

Comment:
It's probably better to interpret this differently to avoid redundancy:

```json
{
    "blacklist": [
        "bl",
        "whatever"
    ]
}
```

Reply:
Changed the format as suggested